ddtrace/internal/compat.py

```diff
@@ -365,3 +368,83 @@ def ip_is_global(ip):
         return parsed_ip.is_global
 
     return not (parsed_ip.is_loopback or parsed_ip.is_private)
+
+
+# https://stackoverflow.com/a/19299884
+class TemporaryDirectory(object):
```

Comment:
If we are only enabling code coverage for Python 3, should we just do the same for git packfile uploading instead of adding this code to make it PY2 compatible? I don't see much value in adding this if ultimately ITR is dependent on both code coverage and git packfile uploading and therefore won't work in Python 2.

Reply:
Personally I don't think so, because keeping Py2 support in this change makes life easier for our future selves when we decide to figure out intelligent test running on Py2.

Reply:
Fair point.